CrewChiefV4/R3E/R3EGameStateMapper.cs

```diff
@@ -1681,6 +1685,7 @@ private TyreCondition getTyreCondition(float percentWear)
                 }
             }
 
+            Boolean isPlayerCarClass = CarData.IsCarClassEqual(opponentData.CarClass, playerCarClass);
```

Comment:
Come to think of it, should we make isPlayerCarClass part of opponentData object ?, this way we wont have to do lookups in many cases where we use IsCarClassEqual().

Reply:
probably. Might be a job for the bigger TimingData refactor (which scares me)